### DISCUSSION TITLE
Being able to read mathematical expressions in the readme file

### DISCUSSION THREAD

**Discussion by Angelk90** [community/community#35178] [Mobile]
It would be nice that through a mobile application on android/ios to be able to read mathematical expressions.

![20221003_153359.jpg](https://user-images.githubusercontent.com/20476002/194015249-04f8b08c-4d55-435f-aab1-3da5c9764758.jpg)

**Comment by jfullstackdev** [comment]
if that is not supported in the Mobile App, what you can do now, if you are really using Mobile to read these files, is to log in to GitHub Web at the browser of your mobile and it should be working
